pss/forwarding_test.go

```diff
@@ -32,6 +32,7 @@ var testCases []testCase
 // selects the peers for message forwarding, depending on the message address
 // and kademlia constellation.
 func TestForwardBasic(t *testing.T) {
+	t.Skip("Flaky on macOS on local machines")
```

Comment:
Based on the skip message, can we skip it only if GOOS is darwin?

Reply:
I have no idea if this works fine on other platforms. If it is indeed a macOS issue, then sure, but I don't know if this is the case.

Reply:
OK, sorry, a precise message gave me the impression that is the problem only on macOS, my mistake...